src/dftbp/api/mm/capi.F90

```diff
@@ -482,6 +484,43 @@ subroutine c_DftbPlus_getGrossCharges(handler, atomCharges)&
 
   end subroutine c_DftbPlus_getGrossCharges
 
+  !> Obtain CM5 charges
+  subroutine c_DftbPlus_getCM5Charges(handler, inputHandler, atomCharges)&
+      & bind(C, name='dftbp_get_cm5_charges')
```

Comment:
As others noted (thanks!), this is something we should discuss in detail as it breaks the logic of the current API. The current logic is to build the input structure (from file or in Fortran also via API), use it to initialize the DFTB+ calculator, and then destroy it. 

Miguel (@steinmig), is there any reason, why we can not expect the consumer of our API to set up the need (and the parameters) for the CM5 charges already in the input? Is there any scenario you can think about, where the possibility of an "on-the-fly" decisions after the initialization is really needed?

Reply:
@aradi Sorry I forgot to write here again, this is finished from my side